examples/tutorial/tutorial_notebooks/introduction/omnisci_on_native/exercise_1.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
We should put an env file with all dependencies required omnisci notebook in omnisci_on_native folder.

Reply:
@YarShev I added this file